scripts/linux/wayland_enable.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+################################################################################
+# This script will enable wayland windowing system
+################################################################################
+
+if [[ "${UID}" != 0 ]]; then
+    (>&2 echo "Error:  $0 must be run as root")
+    exit 1
+fi
+
+enable_wayland() {
+    case ${linuxType} in
+        ubuntu|\"rocky\"|\"centos\")
```

Comment:
why we differentiating between these and debian ?

Reply:
the files in which we need to make the change are different for ubuntu|\"rocky\"|\"centos\" (/etc/gdm3/**_custom.conf_**) and debian (/etc/gdm3/**_daemon.conf_**).